docs/sources/k6-studio/set-up/settings.md

```diff
@@ -0,0 +1,38 @@
+---
+title: 'Settings'
+description: 'Learn how to configure k6 Studio settings'
+weight: 200
+---
+
+# Settings
+
+k6 Studio includes a few settings you can configure by clicking on **Settings** on the bottom-left side of the application.
+
+## Recorder settings
+
+### Automatically detect browser
+
+When this setting is enabled, k6 Studio looks for Google Chrome in the default installation path in your system.
+
+You can turn off this setting and provide a custom path to a Google Chrome executable file.
+
+## Proxy settings
+
+### Port number
+
+The port number that the proxy recorder should listen to. By default, it's set to `6000`. You can customize this value and turn on or off the option to allow k6 Studio to find a different port if the one that's specified is in use.
+
+### Proxy mode
+
+There are two configuration options for proxy mode:
+
+- **Regular**: all requests are performed from the system k6 Studio is running from. This is the default value.
+- **Upstream**: all requests are forwarded to an upstream server.
+
+If you choose the **Upstream** mode, you have to configure a **Server URL** value, and you can optionally provide authentication credentials and a certificate file.
```

Comment:
@Llandy3d @tom-miseur this might be for after the release, but I think it would be nice if we could explain here why someone might want to use the Upstream option. What do you think? 🤔